frontend/packages/knative-plugin/src/components/add/EventSourceForm.tsx

```diff
@@ -28,27 +29,31 @@ const EventSourceForm: React.FC<FormikProps<FormikValues> & OwnProps> = ({
   dirty,
   namespace,
   projects,
-}) => (
-  <Form className="co-deploy-image" onSubmit={handleSubmit}>
-    <EventSourcesSelector eventSourceList={useEventSourceList(namespace)} />
-    {values.type === EventSources.CronJobSource && <CronJobSection />}
-    {values.type === EventSources.SinkBinding && <SinkBindingSection />}
-    {values.type === EventSources.ApiServerSource && <ApiServerSection namespace={namespace} />}
-    <SinkSection namespace={namespace} />
-    <AppSection
-      project={values.project}
-      noProjectsAvailable={projects?.loaded && _.isEmpty(projects.data)}
-    />
-    <FormFooter
-      handleReset={handleReset}
-      errorMessage={status && status.submitError}
-      isSubmitting={isSubmitting}
-      submitLabel="Create"
-      sticky
-      disableSubmit={!dirty || !_.isEmpty(errors)}
-      resetLabel="Cancel"
-    />
-  </Form>
-);
+}) => {
+  return (
```

Comment:
I see this else where in your pr too. Please address all. Prefer immediate return values with no return statement as was present before your change.

Reply:
done

Reply:
I still see that you have explicit return instead of directly returning the JSX.